tests/test_server.py

```diff
@@ -17,6 +27,27 @@ def test_index(self):
         self.assertEqual(data['description'], 'This is the REST API Service for the shopcarts.')
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
-        
+    def test_get_shopcart(self):
+        """ Test Get a Shopcart """
+        # Test for existing shopcarts
+        resp = self.app.get('/shopcarts/1')
+        self.assertEqual( resp.status_code, HTTP_200_OK )
+        data = json.loads(resp.data.decode('utf8'))
+        self.assertEqual (data['uid'], 1)
+        self.assertEqual (data['products'], {})
+
+        resp = self.app.get('/shopcarts/2')
```

Comment:
Only need to do one of these as you're testing the same thing again with no added benefit.

Reply:
Maybe have the second have products instead.